### PR TITLE
Fixed typo mistakes

### PR DESCRIPTION
**Pull request by jignesh-baldha**
### Description
I've fixed typo mistakes.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @jignesh-baldha. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation

**Comment by jignesh-baldha** [comment]
@magento-engcom-team Don't need the test instance, I've just fixed typo mistakes.

File: dev/tests/integration/testsuite/Magento/Cms/Controller/Adminhtml/Wysiwyg/Images/OnInsertTest.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ class OnInsertTest extends AbstractBackendController
      * @magentoConfigFixture admin_store cms/wysiwyg/use_static_urls_in_catalog 1
      * @return void
      */
-    public function testExecuteWhithStaticUrls()
+    public function testExecutewithStaticUrls()
```
Comment (dverkade) [inline dev/tests/integration/testsuite/Magento/Cms/Controller/Adminhtml/Wysiwyg/Images/OnInsertTest.php:26]: Please use camel case, so this should be: testExecuteWithStaticUrls

File: dev/tests/integration/testsuite/Magento/Cms/Controller/Adminhtml/Wysiwyg/Images/OnInsertTest.php [hunk 2]
```diff
@@ -36,7 +36,7 @@ public function testExecuteWhithStaticUrls()
      * @magentoConfigFixture admin_store cms/wysiwyg/use_static_urls_in_catalog 0
      * @return void
      */
-    public function testExecuteWhithoutStaticUrls()
+    public function testExecutewithoutStaticUrls()
```
Comment (dverkade) [inline dev/tests/integration/testsuite/Magento/Cms/Controller/Adminhtml/Wysiwyg/Images/OnInsertTest.php:39]: Please use camel case, so this should be: testExecuteWithoutStaticUrls

**Review by dverkade — CHANGES_REQUESTED** [review]
Hi, please check my comments for this PR.

**Comment by sidolov** [comment]
Hi @jignesh-baldha , unfortunately, we can't accept such Pull Requests to 2.2 release line, all code cleanup like typos, spelling mistakes, etc should be delivered to 2.3 release line. For more information, please, refer to [Magento Code Contributions Guide](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#rules)

**Comment by dverkade** [comment]
I have created a PR for Magento 2.3 here: #17293

**Pull request closed without merging**